internal/orchestrion/context_test.go

```diff
@@ -0,0 +1,107 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package orchestrion
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+type key string
```

Comment:
This is used to bypass the checks from reviewdogs which raised the other comments in this PR